bff-nestjs-service/src/services/cache.ts

```diff
@@ -0,0 +1,17 @@
+import * as moment from 'moment';
```

Comment:
Not an error, but I suppose you can skip the usage of `moment` library. Cause it’s overhead solution for this functionality.